module/Application/view/application/ticket/open.phtml

```diff
@@ -0,0 +1,33 @@
+<h1>Open new ticket</h1>
+
+<?php
+$form = $this->form;
+$form->setAttribute('action', $this->url('register-ticket'));
+
+echo $this->form()->openTag($form);
```

Comment:
Unless this form really looks bad without all this additional markup, you can just use `<?= $this->form($form); ?>`